### ISSUE TITLE
DietPi-Software | Sonarr: NZBDrone Error 100 

### ISSUE THREAD

**Issue by sunnythaper** [MichaIng/DietPi#1277]
#### Required Information:
- DietPi Version | ```cat /DietPi/dietpi/.version```?
158

- SBC Device (EG: RPi 3)?
Odroid C2

- Power supply used (EG: 5v 1A RAVpower)?
Manufacturer supplied power supply

- SD card used (EG: Sandisk ultra)?
Sandisk Ultra 32gig

- Distro (EG: Jessie) | ```uname -a```?
Linux PlexServer 3.14.79+ #1 SMP PREEMPT Sun Jul 23 00:04:44 CEST 2017 aarch64 GNU/Linux

#### Additional Information (if applicable):
- Software title?
Sonarr

- Can this issue be replicated on a fresh installation of DietPi?
Unknown, wasn't willing to try. I do have other Dietpi services that do currently operate quite well for me.

#### Expected behaviour:
Upon "dietpi-software install 144" I should be able to complete the install.

#### Actual behaviour:
Error code 100 saying NZBdrone cannot be found.

#### Steps to reproduce:
dietpi-software install 144

#### Did you submit a ```dietpi-bugreport```?
24ca1211-e327-4b84-852d-050db039b816-0  

#### Extra details:
Other than standard Dietpi-software installs, I have left this unit alone completely. I was looking for the easiest no fuss solution so I didn't exactly go out of my way to do anything non-standard.

**Comment by opnwrt4lede** [comment]
Just to confirm. Have the exactly same Error message (Error 100). My SBC Device is a OPiPC2.

Tried two times - same result:

                                                           │ Apt-get has failed:                                                │                                                           
                                                          │  - Apt string = nzbdrone                                           │                                                           
                                                          │  - Error code = 100                                                │                                                           
                                                          │  - Software Title = Sonarr: automatically download TV shows        │                                                           
                                                          │  - Apt log = /var/log/dietpi-software_apt.log                      │                                                           
                                                          │                                                                    │                                                           
                                                          │ Please report this issue to DietPi (INCLUDING THE APT LOG) so it   │                                                           
                                                          │ can be investigated.                                               │                                                           
                                                          │                                                                    │                                                           
                                                          │ DietPi-Software will now exit.    


```
E: Unable to locate package nzbdrone

 [Info] Please try:
 [Info] - Fixing apt package errors with 'apt-get install -f'
 [Info] - Change apt mirror: 'dietpi-config' > 'Networking: NAS/MISC' > 'Apt Mirror'
 [Info] - Clear and refresh apt 'apt-get update'
 [Info] - If problems persist, please create a Git ticket so DietPi can investigate the issue:  - https://github.com/Fourdee/DietPi/issues
```

It tried the suggestions but no luck.

**Comment by sunnythaper** [comment]
@opnwrt4lede thanks for confirming! I've tried the suggestions as well and still have the issue :/

**Comment by Fourdee** [comment]
@sunnythaper 

Thanks for the report.

Just ran a quick test on:
- 🈯️ Odroid C2 + Stretch, working fine
- 🈯️ XU4 + Jessie fine.

Unable to replicate on fresh installation.

I'll nudge this to v160 for further investigations, and check the bug report. As I am unable to replicate on fresh install, I can only suggest a quick solution is to try the same in the mean time.

**Comment by sunnythaper** [comment]
Thank you, @Fourdee !

**Comment by Fourdee** [comment]
@sunnythaper 

Apologies for the delay, system looks fine, apt source added correctly. Can you try running:
```
dpkg --add-architecture armhf
dietpi-software install 144
```

**Comment by sunnythaper** [comment]
@Fourdee apologize but since this post, I have switched away from DietPi and am just running the individual apps I needed as Docker containers on my oDroid. I guess I wanted Sonarr that badly :P

**Comment by Fourdee** [comment]
@sunnythaper 

No worries.

Confident ```dpkg --add-architecture armhf``` should resolve this. Passed testing. Marking as fixed in v6.0.
